Throwings.h

```diff
@@ -0,0 +1,51 @@
+﻿#pragma once
+
+#ifndef THROWINGS_H
+#define THROWINGS_H
+
+#include <SFML/Graphics.hpp>
+#include <SFML/Audio.hpp>
+#include <string>
+#include <cassert>
+#include <iostream>
+#include "ScoreAndLife.h"
+
+using namespace sf;
+using namespace std;
+
+class Throwings {
+private:
+    float x;
+    float speed;
+    Texture food_img;
+    Texture ddong_img;
+    Sprite food;
+    Sprite ddong;
+    Clock clock;
+    bool is_food;
+
+public:
+    bool hit = false;
+    bool destroyed = false;
+    bool destroyed_hit = false;
+    Throwings(float _x, float _speed, bool _is_food)
+        : x(_x), speed(_speed), is_food(_is_food)
+    {
+        if (this->is_food == true) {
```

Comment:
```suggestion
        if (this->is_food) {
```
로 더 간단히 써주실 수도 있을 것 같습니다!